### DISCUSSION TITLE
How do I display folder icons correctly?

### DISCUSSION THREAD

**Discussion by rasivasu** [mcchrish/nnn.vim#157] [Q&A]
I have folder icons enabled by compiling `make O_NERD=1`.  But these icons are not displayed correctly when using nnn.vim.  Can you please tell me the setting to be used to correct this?

**Comment by mcchrish** [comment]
Please see this thread https://github.com/mcchrish/nnn.vim/discussions/110#discussioncomment-1210181
Might be the same issue.

**Comment by rasivasu** [comment]
@mcchrish No, it is not.  The icons are not displayed when I typed ":terminal nnn".  

However I am facing the issue only within Kitty (https://sw.kovidgoyal.net/kitty/) and not in iterm.

I am having the following line in kitty.conf.
`font_family      FiraCode Nerd Font Mono`

and following is the output of `kitty +list-fonts --psnames | grep Fira`

> Fira Code
>     Fira Code Bold (FiraCodeRoman-Bold)
>     Fira Code Light (FiraCode-Light)
>     Fira Code Medium (FiraCodeRoman-Medium)
>     Fira Code Regular (FiraCodeRoman-Regular)
>     Fira Code SemiBold (FiraCodeRoman-SemiBold)
> FiraCode Nerd Font Mono
>     FiraCode Nerd Font Mono Bold (FiraCodeNerdFontCompleteM-Bold)
>     FiraCode Nerd Font Mono Light (FiraCodeNerdFontCompleteM-Light)
>     FiraCode Nerd Font Mono Medium (FiraCodeNerdFontCompleteM-Medium)
>     FiraCode Nerd Font Mono Regular (FiraCodeNerdFontCompleteM-Regular)
>     FiraCode Nerd Font Mono Retina (FiraCodeNerdFontCompleteM-Retina)

**Comment by rasivasu** [comment]
Attaching the screenshot of neovim with both NERDTree and nnn plugins for reference.

![image](https://user-images.githubusercontent.com/1153582/158809857-a7eccd2f-8520-4e90-8bfb-c28f6ca9a80a.png)
